### DISCUSSION TITLE
`Issues` button on main page of the Android app to keep sorting settings

### DISCUSSION THREAD

**Discussion by Crabstove** [community/community#4715] [Mobile] [answered]
So for me it would be ideal if I could save settings to see issues that were assigned to me by default instead of issues that I created. As it is right now, every time I click on `Issues` I do not see what Issues are assigned to me currently and I have to sort it manually to show "assigned only" instead of "created only".

 I don't really create any issues on my company's GitHub, my senior does and then he assignes them to me or other developers. So currently if I want to conveniently check my tasks from my phone I have to do extra clicks no matter what.

**Comment by brianlovin** [comment, marked as answer]
@Crabstove thanks for the note here – we agree. We're investigating how we can persist filters across sessions, and will try to make this happen soon!
